### PR TITLE
fix: listing history tweaks

### PR DESCRIPTION
**Pull request by nickreynolds**

### CODE REVIEW

File: packages/components/src/ChallengeResultsChart/styledComponents.tsx [hunk 1]
```diff
@@ -72,7 +74,7 @@ export const VotesPerTokenVote = styled.div<VotesPerTokenVoteProps>`
         ? colors.accent.CIVIL_TEAL
         : colors.accent.CIVIL_RED};
     margin-right: 4px;
-  ]
+  }
```
Comment (tobek) [inline packages/components/src/ChallengeResultsChart/styledComponents.tsx]: :exclamation: good catch. It's a bummer that we don't (can't?) have linting inside styled components. I've seen pretty often for instance a missing semicolon that silently borks the rules before and after.

**Comment by dankins** [comment]
:tada: This PR is included in version 4.9.3 :tada:

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Pull request merged**